File: Desenvolvimento/server/src/ApplicationCore/Entities/TramiteArquivo.cs
using ApplicationCore.Interfaces.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationCore.Entities
{
    public class TramiteArquivo : IBaseEntity<long>
    {
        public long Id { get; set; }
        public long TramiteId { get; set; }
        public virtual Tramite Tramite { get; set; }
        public long ArquivoId { get; set; }
        public virtual Arquivo Arquivo { get; set; }
        public long? FluxoItemTipoAnexoId { get; set; }
        public FluxoItemTipoAnexo FluxoItemTipoAnexo { get; set; }
    }
}

File: Desenvolvimento/client/src/@fuse/types/models/tramite-arquivo.ts
import { Tramite } from './tramite';
import { Arquivo } from './arquivo';
import { FluxoItemAnexo } from './fluxo-item-anexo';
import { BaseModel } from '../base-model';

export class TramiteArquivo extends BaseModel {
    constructor(
        public id: number,
        public tramiteId?: number,
        public arquivoId?: number,
        public fluxoItemTipoAnexoId?: number,
        public tramite?: Tramite,
        public arquivo?: Arquivo,
        public fluxoItemTipoAnexo?: FluxoItemAnexo,
    ) { super(); }
}
